src/snyk/snykCode/views/issueTreeProvider.ts

```diff
@@ -150,4 +154,20 @@ export class IssueTreeProvider extends AnalysisTreeNodeProvder {
   protected getIssueDescriptionText(dir: string | undefined, diagnostics: readonly Diagnostic[]): string | undefined {
     return `${dir} - ${diagnostics.length} issue${diagnostics.length === 1 ? '' : 's'}`;
   }
+
+  protected getFilteredIssues(diagnostics: readonly Diagnostic[]): Diagnostic[] {
+    return diagnostics.filter(issue => {
+      switch (issue.severity) {
+        case DiagnosticSeverity.Error:
+          return this.configuration.severityFilter.high;
+        case DiagnosticSeverity.Warning:
+          return this.configuration.severityFilter.medium;
+        case DiagnosticSeverity.Information:
+        case DiagnosticSeverity.Hint:
+          return this.configuration.severityFilter.low;
+        default:
+          return true;
```

Comment:
should we prepare for `critical` level issues? afaik it will be introducing in Snyk code soon.

Reply:
Let's address it when it comes. It is not a scope of this PR.

Reply:
(There are more changes than just this place to be edited for that).